docs/src/main/paradox/release-notes/releases-1.1.md

```diff
@@ -1,9 +1,8 @@
 # Release Notes (1.1.x)
 
-## 1.1.0-M1
+## 1.1.0
 
-Release notes for Apache Pekko Persistence JDBC 1.1.0-M1. See [GitHub Milestone](https://github.com/apache/pekko-persistence-jdbc/milestone/1?closed=1) for fuller list of changes.
-As with all milestone releases, this release is not recommended for production use - it is designed to allow users to try out the changes in a test environment.
+Release notes for Apache Pekko Persistence JDBC 1.1.0. See [GitHub Milestone](https://github.com/apache/pekko-persistence-jdbc/milestone/1?closed=1) for a fuller list of changes.
```

Comment:
I guess https://github.com/apache/pekko-persistence-jdbc/milestone/3?closed=1 is not worth mentioning :+1:

Reply:
there isn't anything other than release related changes in the 2nd milestone